gradle.properties

```diff
@@ -14,16 +14,13 @@
 # limitations under the License.
 #
 version=0.11.2
-ideaVersion=2018.1.8
+ideaVersion=PC-2021.2.3
```

Comment:
Should I rename this to `ideVersion`? Since it's no longer IDEA-specific.

Reply:
Yeah sure, good idea (pun intended).